metaStoreApp/app/controllers/api/v1/audios_controller.rb

```diff
@@ -1,6 +1,42 @@
 module Api
   module V1
     class AudiosController < MediasController
+      # creates a new audio
+      api :POST, '/audios/', "Creates a new audio"
```

Comment:
why are we having another controller for audio? from a API point of view, audio and video have same set of functionalities

Reply:
Since duration_tc have different functionalities, i have implemented different controller for audio and video. The modules that can be reused in subclasses, i have reused them and differentiated the json format rendered as audio or video in the response

Reply:
duration_tc is implemented in the model, why controller needs to worry about that?

Reply:
I used them to differentiate the render response message for audio as "saved audio", and for video as "saved video". I could put this in the media itself instead and can check based on media_type passed.